README.ua.md

```diff
@@ -0,0 +1,350 @@
+<br>
+
+<div align="center">
+  <a href="https://hydralauncher.site">
+    <img src="./resources/icon.png" width="144"/>
+  </a>
+  <h1 align="center">Hydra Launcher</h1>
+  <p align="center">
+    <strong>Hydra - це ігровий лаунчер з власним вбудованим bittorrent-клієнтом і самокерованим збирачем репаків.</strong>
+  </p>
+  <p>
+    <a href="https://discord.gg/hydralauncher">
+      <img src ="https://img.shields.io/discord/1220692017311645737?style=flat&logo=discord&label=Hydra&labelColor=%231c1c1c"/>
+    </a>
+    <a href="https://github.com/hydralauncher/hydra">
+      <img src="https://img.shields.io/github/actions/workflow/status/hydralauncher/hydra/build.yml" />
+    </a>
+    <a href="https://github.com/hydralauncher/hydra">
+      <img src="https://img.shields.io/github/package-json/v/hydralauncher/hydra" />
+    </a>
+  </p>
+
+![Hydra Catalogue](./docs/screenshot.png)
+
+</div>
+
+<br>
+
+### Мова
+[![ua](https://img.shields.io/badge/lang-ua-red)](https://github.com/hydralauncher/hydra/blob/main/README.ru.md)
```

Comment:
This badge is for UA but is linking to RU README. 
Could you fix and add the badges for RU and EN readme as weelll?

Reply:
I fixed the badge for UA. Just a second, I will add RU and EN badges.